src/util_regression.mjs

```diff
@@ -102,7 +102,8 @@ util_regression.KalmanFilter.prototype.update = function(z) {
     var P_p = add(mult(mult(this.F,this.P), transpose(this.F)), this.Q); //Predicted covaraince
 
     //Calculate the update values
-    var y = sub(z, mult(this.H, X_p)); // This is the measurement error (between what we expect and the actual value)
+    // Use `squeeze` to transform the multiplied value to match the measurement's form, i.e. column vector => row vector
+    var y = sub(z, mathjs.squeeze(mult(this.H, X_p))); // This is the measurement error (between what we expect and the actual value)
```

Comment:
`H * X_p` produces a `2*1` column vector, we need to [squeeze](https://mathjs.org/docs/reference/functions/squeeze.html) it to match the input's form.

Reply:
We can reshape `z` to a column vector so that the later transformation of `y` (see below) is no longer needed.
https://github.com/brownhci/WebGazer/blob/7ff29a32b12048362750d0594ecf8375dcdd22a0/src/util_regression.mjs#L111-L114

Reply:
Done.